_15thnight/core.py

```diff
@@ -24,10 +25,10 @@ def send_out_alert(alert_form):
     providers = User.providers_with_services(need_ids)
     for provider in providers:
         needs_provided = [
-            need for need in provider.services if need.id in need_ids
+            need_ for need_ in provider.services if need_.id in need_ids
```

Comment:
This rename is not needed because the previous `need` declaration is in a for loop.